Classes/Repository/RepositoryIssueSummaryModel.swift

```diff
@@ -56,5 +56,6 @@ class RepositoryIssueSummaryModel: ListDiffable {
             && author == object.author
             && created == object.created
             && title.attributedText.string == object.title.attributedText.string
+            && labels == object.labels
```

Comment:
Note that this check is `O(n)` (comparing all contents of an array). I'm going to clean this up to be much faster.